cli/emit.rs

```diff
@@ -453,6 +564,13 @@ pub fn check_and_maybe_emit(
         }
       }
     }
+
+    // now insert these items into the cache
+    for (specifier, data) in emit_data_items.into_iter() {
+      if let Some(cache_data) = data.into_cache_data() {
+        cache.set_emit_data(specifier, cache_data)?;
```

Comment:
In the future this will be a single sqlite db update where it updates a row all at once, so that's why the code was changed to have a single update method.